snippets/common/VS_Snippets_WebNet/CustomServerControlCollectionProperties/projectsample/customservercontrolcollection/samples.aspnet.vb/quickcontacts.vb

```diff
@@ -38,8 +38,8 @@ Namespace Samples.AspNet.VB.Controls
         End Property
 
         ' The contacts are rendered in an HTML table.
-        Protected Overrides Sub RenderContents( _
-        ByVal writer As HtmlTextWriter)
+        Protected Overrides Sub RenderContents(
+        writer As HtmlTextWriter)
```

Comment:
nit: this could be moved to line 41, which makes the method signature more readable.

Reply:
Will be fixed in tonight's commit